src/clos/describe.lisp

```diff
@@ -276,10 +276,12 @@
 ;;; function
 (defmethod describe ((obj function) &optional (stream *standard-output*))
   (let ((name (oget obj "fname"))
+	(lambda-list (oget obj "lambdalist"))
```

Comment:
can we use the same functions we exposed here? `function-lambda-list` ?

Reply:
I'll make the change.